commerce-checkout-web/src/main/resources/META-INF/resources/checkout_step/payment_method.jsp

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
@andrea-ale-sbarra looks like an extra line got in here -- can you remove in a follow-up PR?